info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T04:41:35.532Z"}
info: Connected to the database. {"timestamp":"2022-05-26T04:41:42.071Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T05:13:17.321Z"}
info: Connected to the database. {"timestamp":"2022-05-26T05:13:17.476Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T06:57:11.653Z"}
info: Connected to the database. {"timestamp":"2022-05-26T06:57:11.843Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T07:03:36.900Z"}
info: Connected to the database. {"timestamp":"2022-05-26T07:03:37.077Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected token { in JSON at position 7
    at JSON.parse (<anonymous>)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-26T07:10:03.254Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T07:17:07.649Z"}
info: Connected to the database. {"timestamp":"2022-05-26T07:17:07.887Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T08:49:28.146Z"}
info: Connected to the database. {"timestamp":"2022-05-26T08:49:28.649Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T08:57:58.622Z"}
info: Connected to the database. {"timestamp":"2022-05-26T08:57:58.762Z"}
error: uncaughtException: listen EADDRINUSE: address already in use :::3000
Error: listen EADDRINUSE: address already in use :::3000
    at Server.setupListenHandle [as _listen2] (node:net:1372:16)
    at listenInCluster (node:net:1420:12)
    at Server.listen (node:net:1508:7)
    at Function.listen (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\application.js:635:24)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\/index.js:37:5)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._compile (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Object.newLoader [as .js] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\node\lib\_babel-node.js:176:21)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32) {"date":"Thu May 26 2022 14:27:58 GMT+0530 (India Standard Time)","error":{"address":"::","code":"EADDRINUSE","errno":-4091,"port":3000,"syscall":"listen"},"exception":true,"os":{"loadavg":[0,0,0],"uptime":353981},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19218836,"external":20818065,"heapTotal":78721024,"heapUsed":49363616,"rss":107986944},"pid":404,"uid":null,"version":"v16.15.0"},"stack":"Error: listen EADDRINUSE: address already in use :::3000\n    at Server.setupListenHandle [as _listen2] (node:net:1372:16)\n    at listenInCluster (node:net:1420:12)\n    at Server.listen (node:net:1508:7)\n    at Function.listen (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\express\\lib\\application.js:635:24)\n    at Object.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\/index.js:37:5)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Module._compile (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Object.newLoader [as .js] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at Object.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\@babel\\node\\lib\\_babel-node.js:176:21)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)","timestamp":"2022-05-26T08:57:58.856Z","trace":[{"column":16,"file":"node:net","function":"Server.setupListenHandle [as _listen2]","line":1372,"method":"setupListenHandle [as _listen2]","native":false},{"column":12,"file":"node:net","function":"listenInCluster","line":1420,"method":null,"native":false},{"column":7,"file":"node:net","function":"Server.listen","line":1508,"method":"listen","native":false},{"column":24,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\express\\lib\\application.js","function":"Function.listen","line":635,"method":"listen","native":false},{"column":5,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\/index.js","function":null,"line":37,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":24,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js","function":"Module._compile","line":136,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":7,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js","function":"Object.newLoader [as .js]","line":141,"method":"js]","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":822,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":77,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":21,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\@babel\\node\\lib\\_babel-node.js","function":null,"line":176,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T08:59:05.242Z"}
info: Connected to the database. {"timestamp":"2022-05-26T08:59:05.394Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T09:18:53.652Z"}
info: Connected to the database. {"timestamp":"2022-05-26T09:18:53.811Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T10:28:48.688Z"}
info: Connected to the database. {"timestamp":"2022-05-26T10:28:48.860Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of null (reading 'firstName')
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:16:24)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-26T10:31:58.552Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T10:42:50.674Z"}
info: Connected to the database. {"timestamp":"2022-05-26T10:42:50.866Z"}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:13:07 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":360290},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18557768,"external":20131505,"heapTotal":46653440,"heapUsed":43374304,"rss":77197312},"pid":10360,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:43:07.763Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T10:43:58.326Z"}
info: Connected to the database. {"timestamp":"2022-05-26T10:43:58.469Z"}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:14:04 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":360346},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18596094,"external":20222882,"heapTotal":78635008,"heapUsed":43616328,"rss":107712512},"pid":8436,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:44:04.385Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T10:46:39.584Z"}
info: Connected to the database. {"timestamp":"2022-05-26T10:46:39.735Z"}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:16:49 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":360511},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19665560,"external":25741805,"heapTotal":79683584,"heapUsed":56678800,"rss":109764608},"pid":11196,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:46:49.307Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T10:47:32.011Z"}
info: Connected to the database. {"timestamp":"2022-05-26T10:47:32.162Z"}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:17:39 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":360561},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\build\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18919592,"external":20411455,"heapTotal":38715392,"heapUsed":27658672,"rss":62693376},"pid":8464,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:47:39.361Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T10:54:48.985Z"}
info: Connected to the database. {"timestamp":"2022-05-26T10:54:49.148Z"}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:25:09 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":361012},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\build\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18903648,"external":20412426,"heapTotal":38715392,"heapUsed":27817128,"rss":62320640},"pid":5744,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:55:09.844Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T10:57:14.053Z"}
info: Connected to the database. {"timestamp":"2022-05-26T10:57:14.216Z"}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 16:27:25 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":361147},"process":{"argv":["C:\\Program Files\\nodejs\\node.exe","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\build\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18920827,"external":20417307,"heapTotal":38715392,"heapUsed":27652520,"rss":62607360},"pid":5148,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T10:57:25.265Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T15:03:24.719Z"}
info: Connected to the database. {"timestamp":"2022-05-26T15:03:24.863Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T15:10:16.667Z"}
info: Connected to the database. {"timestamp":"2022-05-26T15:10:16.739Z"}
error: uncaughtException: Cannot read properties of undefined (reading '$elemMatch')
TypeError: Cannot read properties of undefined (reading '$elemMatch')
    at model.Query._castFields (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4981:23)
    at model.Query.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:2240:23)
    at model.Query._wrappedThunk [as _findOne] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\helpers\query\wrapThunk.js:16:8)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\kareem\index.js:370:33
    at processTicksAndRejections (node:internal/process/task_queues:78:11) {"date":"Thu May 26 2022 20:41:36 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":376399},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18473528,"external":20136923,"heapTotal":45604864,"heapUsed":41631864,"rss":75907072},"pid":3704,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Cannot read properties of undefined (reading '$elemMatch')\n    at model.Query._castFields (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:4981:23)\n    at model.Query.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js:2240:23)\n    at model.Query._wrappedThunk [as _findOne] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js:16:8)\n    at C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js:370:33\n    at processTicksAndRejections (node:internal/process/task_queues:78:11)","timestamp":"2022-05-26T15:11:37.112Z","trace":[{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":"model.Query._castFields","line":4981,"method":"_castFields","native":false},{"column":23,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\query.js","function":null,"line":2240,"method":null,"native":false},{"column":8,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\mongoose\\lib\\helpers\\query\\wrapThunk.js","function":"model.Query._wrappedThunk [as _findOne]","line":16,"method":"_wrappedThunk [as _findOne]","native":false},{"column":33,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\kareem\\index.js","function":null,"line":370,"method":null,"native":false},{"column":11,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":78,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T15:13:31.652Z"}
info: Connected to the database. {"timestamp":"2022-05-26T15:13:31.743Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T15:15:09.072Z"}
info: Connected to the database. {"timestamp":"2022-05-26T15:15:09.170Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-26T15:35:09.666Z"}
info: Connected to the database. {"timestamp":"2022-05-26T15:35:09.732Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:06:14.256Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:06:14.332Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:14:41.180Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:14:41.248Z"}
error: 
    status - 500 
    message - Error: Password not match
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:22:13)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-27T03:15:23.682Z"}
error: 
    status - 500 
    message - Error: User does not exist
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:17:11)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-27T03:15:41.383Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:20:36.591Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:20:36.652Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:25:09.434Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:25:09.497Z"}
error: 
    status - 500 
    message - ReferenceError: userData is not defined
    at _callee2$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:10:43)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.registerUser (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:9:26)
    at _callee2$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/user.controller.js:25:36)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:25:14.458Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:26:01.070Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:26:01.153Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:27:48.409Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:27:48.499Z"}
error: 
    status - 500 
    message - Error: User already exist
    at _callee2$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:15:11)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:27:54.735Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:31:27.797Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:31:27.860Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected token , in JSON at position 77
    at JSON.parse (<anonymous>)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:33:52.540Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:37:31.084Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:37:31.322Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:38:22.774Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:38:22.843Z"}
error: 
    status - 500 
    message - ValidationError: "email" must be a valid email 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:38:40.762Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:40:09.586Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:40:09.660Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\validators\/user.validator.js:12:16)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:175:3) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:40:19.154Z"}
error: 
    status - 500 
    message - ReferenceError: HttpStatus is not defined
    at newUserValidator (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\validators\/user.validator.js:12:16)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:175:3)
    at router (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:328:13)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:175:3) 
    url - /api/v1/users/signup 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T03:40:24.374Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:41:33.308Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:41:33.421Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:44:07.296Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:44:07.359Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T03:46:18.276Z"}
info: Connected to the database. {"timestamp":"2022-05-27T03:46:18.341Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T05:26:51.673Z"}
info: Connected to the database. {"timestamp":"2022-05-27T05:26:51.808Z"}
error: 
    status - 500 
    message - Error: Password not match
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:33:13)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-27T05:27:55.333Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T08:19:18.432Z"}
info: Connected to the database. {"timestamp":"2022-05-27T08:19:18.546Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T09:06:23.800Z"}
info: Connected to the database. {"timestamp":"2022-05-27T09:06:23.929Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-27T09:32:09.680Z"}
info: Connected to the database. {"timestamp":"2022-05-27T09:32:10.077Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-28T04:05:42.424Z"}
info: Connected to the database. {"timestamp":"2022-05-28T04:05:42.923Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-28T04:11:47.425Z"}
info: Connected to the database. {"timestamp":"2022-05-28T04:11:47.618Z"}
error: 
    status - 500 
    message - Error: Password not match
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:30:13)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-28T04:11:53.306Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T03:30:03.746Z"}
info: Connected to the database. {"timestamp":"2022-05-30T03:30:03.866Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T03:39:32.174Z"}
info: Connected to the database. {"timestamp":"2022-05-30T03:39:32.275Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T03:42:16.110Z"}
info: Connected to the database. {"timestamp":"2022-05-30T03:42:16.180Z"}
error: 
    status - 500 
    message - Error: secretOrPrivateKey must have a value
    at Object.sign (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\jsonwebtoken\sign.js:107:20)
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:29:23)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T03:42:32.717Z"}
error: 
    status - 500 
    message - Error: secretOrPrivateKey must have a value
    at Object.sign (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\jsonwebtoken\sign.js:107:20)
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:29:23)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T03:43:30.593Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T03:44:14.282Z"}
info: Connected to the database. {"timestamp":"2022-05-30T03:44:14.374Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T03:55:49.203Z"}
info: Connected to the database. {"timestamp":"2022-05-30T03:55:49.263Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T03:58:25.471Z"}
info: Connected to the database. {"timestamp":"2022-05-30T03:58:25.555Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T04:02:29.039Z"}
info: Connected to the database. {"timestamp":"2022-05-30T04:02:29.106Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T04:03:24.341Z"}
info: Connected to the database. {"timestamp":"2022-05-30T04:03:24.465Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T04:41:05.361Z"}
info: Connected to the database. {"timestamp":"2022-05-30T04:41:05.470Z"}
error: 
    status - 500 
    message - Error: User does not exist
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/user.service.js:25:11)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T04:41:20.150Z"}
error: uncaughtException: Router.use() requires a middleware function but got a Object
TypeError: Router.use() requires a middleware function but got a Object
    at Function.use (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:469:13)
    at routes (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\routes\/index.js:16:10)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\/index.js:32:32)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._compile (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Object.newLoader [as .js] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\node\lib\_babel-node.js:176:21)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12) {"date":"Mon May 30 2022 10:14:24 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":684366},"process":{"argv":["node","C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\index.js"],"cwd":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":19041760,"external":20636006,"heapTotal":79712256,"heapUsed":54380144,"rss":108691456},"pid":3920,"uid":null,"version":"v16.15.0"},"stack":"TypeError: Router.use() requires a middleware function but got a Object\n    at Function.use (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\express\\lib\\router\\index.js:469:13)\n    at routes (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\routes\\/index.js:16:10)\n    at Object.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\/index.js:32:32)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Module._compile (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Object.newLoader [as .js] (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at Object.<anonymous> (C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\@babel\\node\\lib\\_babel-node.js:176:21)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:822:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)","timestamp":"2022-05-30T04:44:24.189Z","trace":[{"column":13,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\express\\lib\\router\\index.js","function":"Function.use","line":469,"method":"use","native":false},{"column":10,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\routes\\/index.js","function":"routes","line":16,"method":null,"native":false},{"column":32,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\src\\/index.js","function":null,"line":32,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":24,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js","function":"Module._compile","line":136,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":7,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\pirates\\lib\\index.js","function":"Object.newLoader [as .js]","line":141,"method":"js]","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":822,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":77,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":21,"file":"C:\\Users\\user\\Desktop\\FundooNotes\\FundooNotes\\node_modules\\@babel\\node\\lib\\_babel-node.js","function":null,"line":176,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":822,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":77,"method":"executeUserEntryPoint [as runMain]","native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T04:45:06.050Z"}
info: Connected to the database. {"timestamp":"2022-05-30T04:45:06.162Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T06:35:17.747Z"}
info: Connected to the database. {"timestamp":"2022-05-30T06:35:17.830Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T06:38:35.547Z"}
info: Connected to the database. {"timestamp":"2022-05-30T06:38:35.610Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T06:40:09.947Z"}
info: Connected to the database. {"timestamp":"2022-05-30T06:40:10.020Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T06:47:50.093Z"}
info: Connected to the database. {"timestamp":"2022-05-30T06:47:50.184Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T06:49:53.619Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T06:50:14.374Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-30T06:50:25.262Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-30T06:50:38.160Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/users/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-30T06:50:44.699Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:4:27)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.createNote (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:4:24)
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:6:38)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T07:11:44.947Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:4:27)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.createNote (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:4:24)
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:6:38)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/notes 
    method - POST 
    IP - ::1
   {"timestamp":"2022-05-30T08:09:27.494Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T08:10:16.661Z"}
info: Connected to the database. {"timestamp":"2022-05-30T08:10:16.848Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T08:54:44.597Z"}
info: Connected to the database. {"timestamp":"2022-05-30T08:54:44.655Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T08:56:22.117Z"}
info: Connected to the database. {"timestamp":"2022-05-30T08:56:22.207Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T08:56:40.759Z"}
info: Connected to the database. {"timestamp":"2022-05-30T08:56:41.080Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T08:56:58.918Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T08:57:08.388Z"}
info: Connected to the database. {"timestamp":"2022-05-30T08:57:08.490Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T08:58:44.851Z"}
info: Connected to the database. {"timestamp":"2022-05-30T08:58:44.954Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T08:59:04.981Z"}
info: Connected to the database. {"timestamp":"2022-05-30T08:59:05.066Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T08:59:23.457Z"}
info: Connected to the database. {"timestamp":"2022-05-30T08:59:23.564Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:00:41.473Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:00:41.551Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:00:53.666Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:00:53.758Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:01:14.227Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:01:14.333Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:04:12.312Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:04:12.412Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:05:07.856Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:05:07.954Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:06:13.007Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:06:13.074Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:07:39.805Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:07:39.908Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:08:04.096Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:08:04.187Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:11:32.045Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:11:32.117Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:13:31.246Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:13:31.316Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:15:03.093Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:15:03.193Z"}
error: 
    status - 500 
    message - ReferenceError: UserService is not defined
    at _callee3$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:30:27)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at updateNote (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:30:24)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:284:15
    at param (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:365:14) 
    url - /api/v1/notes/62947bfc2d56650e080715d4 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-05-30T09:23:26.458Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:24:28.660Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:24:28.753Z"}
error: 
    status - 500 
    message - ReferenceError: User is not defined
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:27:27)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.deleteUser (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:27:24)
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:45:25)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/notes/62947bfc2d56650e080715d4 
    method - DELETE 
    IP - ::1
   {"timestamp":"2022-05-30T09:26:00.612Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T09:26:25.611Z"}
info: Connected to the database. {"timestamp":"2022-05-30T09:26:25.687Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T14:58:22.785Z"}
info: Connected to the database. {"timestamp":"2022-05-30T14:58:22.976Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T15:55:43.401Z"}
info: Connected to the database. {"timestamp":"2022-05-30T15:55:43.484Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T15:56:46.768Z"}
info: Connected to the database. {"timestamp":"2022-05-30T15:56:46.895Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-30T15:57:03.305Z"}
info: Connected to the database. {"timestamp":"2022-05-30T15:57:03.407Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T01:26:32.164Z"}
info: Connected to the database. {"timestamp":"2022-05-31T01:26:32.367Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes/62956fdc075c272174a262fb 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-31T02:21:44.867Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:160:10)
    at parse (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\types\json.js:83:15)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:231:16)
    at done (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/v1/notes/62956fdc075c272174a262fb 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-31T02:22:01.096Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T02:50:40.223Z"}
info: Connected to the database. {"timestamp":"2022-05-31T02:50:40.282Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T02:51:38.493Z"}
info: Connected to the database. {"timestamp":"2022-05-31T02:51:38.603Z"}
error: 
    status - 500 
    message - ReferenceError: UserService is not defined
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:4:28)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
    at getAllNotes (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:4:25)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:144:13)
    at Route.dispatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\route.js:114:3)
    at Layer.handle [as handle_request] (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\express\lib\router\index.js:346:12) 
    url - /api/v1/notes/ 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-31T02:52:04.674Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T02:52:57.291Z"}
info: Connected to the database. {"timestamp":"2022-05-31T02:52:57.389Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:21:37.739Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:21:37.839Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:23:02.254Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:23:02.423Z"}
error: 
    status - 500 
    message - ReferenceError: cosole is not defined
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\middlewares\/auth.middleware.js:23:5)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-31T03:23:17.986Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:24:05.964Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:24:06.060Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:25:24.321Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:25:24.386Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:26:07.182Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:26:07.290Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:27:42.370Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:27:42.471Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:28:51.017Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:28:51.112Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:40:32.316Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:40:32.448Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:42:58.421Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:42:58.531Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:43:51.699Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:43:51.781Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:44:25.135Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:44:25.310Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:44:55.871Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:44:55.957Z"}
error: 
    status - 500 
    message - CastError: Cast to string failed for value "{ UserID: 'kalir888@gmail.com' }" (type Object) at path "UserID" for model "Note"
    at model.Query.exec (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-31T03:45:12.352Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:46:34.355Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:46:34.484Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:46:51.680Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:46:51.781Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T03:47:33.571Z"}
info: Connected to the database. {"timestamp":"2022-05-31T03:47:33.638Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:04:29.923Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:04:30.000Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:05:26.103Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:05:26.199Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:07:02.372Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:07:02.472Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:07:55.288Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:07:55.372Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:08:23.782Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:08:23.887Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:09:29.918Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:09:30.010Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:10:10.170Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:10:10.287Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:10:51.360Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:10:51.454Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:11:37.214Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:11:37.289Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:34:51.460Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:34:51.546Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:43:57.972Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:43:58.068Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:44:24.805Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:44:24.894Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:45:02.430Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:45:02.523Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:45:47.547Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:45:47.637Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:46:08.617Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:46:08.716Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:46:28.761Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:46:28.835Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:54:26.491Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:54:26.564Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T04:54:43.215Z"}
info: Connected to the database. {"timestamp":"2022-05-31T04:54:43.277Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:10:28.176Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:10:28.280Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:11:42.275Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:11:42.357Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:13:33.731Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:13:33.813Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:14:02.438Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:14:02.526Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:17:30.025Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:17:30.140Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:22:31.339Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:22:31.401Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:22:55.987Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:22:56.063Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:23:05.318Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:23:05.399Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:23:54.182Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:23:54.254Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:24:04.735Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:24:04.805Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:25:07.496Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:25:07.566Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:25:37.540Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:25:37.628Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:53:08.478Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:53:08.571Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:53:16.451Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:53:16.591Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:56:21.564Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:56:21.736Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:57:30.988Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:57:31.086Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T05:57:51.025Z"}
info: Connected to the database. {"timestamp":"2022-05-31T05:57:51.112Z"}
error: 
    status - 500 
    message - ReferenceError: K is not defined
    at _callee$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:5:41)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/notes 
    method - GET 
    IP - ::1
   {"timestamp":"2022-05-31T06:10:10.974Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T06:11:00.940Z"}
info: Connected to the database. {"timestamp":"2022-05-31T06:11:01.013Z"}
error: 
    status - 500 
    message - ReferenceError: body is not defined
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:21:7)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.updateNote (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:19:24)
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:46:38)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/notes/6295b2aee3ca8022fcecba63 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-05-31T06:18:52.879Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T06:19:51.887Z"}
info: Connected to the database. {"timestamp":"2022-05-31T06:19:51.956Z"}
error: 
    status - 500 
    message - ReferenceError: body is not defined
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:21:7)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12)
    at Object.updateNote (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\services\/note.service.js:19:24)
    at _callee4$ (C:\Users\user\Desktop\FundooNotes\FundooNotes\src\controllers\/note.controller.js:46:38)
    at tryCatch (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\user\Desktop\FundooNotes\FundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9) 
    url - /api/v1/notes/6295b2aee3ca8022fcecba63 
    method - PUT 
    IP - ::1
   {"timestamp":"2022-05-31T06:19:56.437Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T06:20:45.309Z"}
info: Connected to the database. {"timestamp":"2022-05-31T06:20:45.397Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-05-31T06:23:12.086Z"}
info: Connected to the database. {"timestamp":"2022-05-31T06:23:12.177Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-01T03:21:23.359Z"}
info: Connected to the database. {"timestamp":"2022-06-01T03:21:23.566Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/notes 
      method - POST 
      IP - ::1
     {"timestamp":"2022-06-01T03:46:39.717Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-01T03:52:56.483Z"}
info: Connected to the database. {"timestamp":"2022-06-01T03:52:56.704Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-01T03:54:20.701Z"}
info: Connected to the database. {"timestamp":"2022-06-01T03:54:20.821Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-01T03:55:09.231Z"}
info: Connected to the database. {"timestamp":"2022-06-01T03:55:09.347Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-06-01T04:07:10.390Z"}
info: Connected to the database. {"timestamp":"2022-06-01T04:07:10.531Z"}
